adjust-bttn-size.js

```diff
@@ -1,19 +1,19 @@
 function changeButtonSize() {
-    // Get the selected size from the dropdown
-    const size = document.getElementById("button-size").value;
-    
-    // Get all buttons on the page
-    const buttons = document.querySelectorAll("button");
+  // Get the selected size from the dropdown
+  const size = document.getElementById("button-size").value;
+  
+  // Get all buttons on the page
+  const buttons = document.querySelectorAll("button");
 
-    // Remove any existing size classes
-    buttons.forEach(button => {
-        button.classList.remove("button-medium", "button-large");
-        
-        // Apply the selected size class
-        if (size === "medium") {
-            button.classList.add("button-medium");
-        } else if (size === "large") {
-            button.classList.add("button-large");
-        }
-    });
-}
+  // Remove any existing size classes
+  buttons.forEach(button => {
+      button.classList.remove("button-medium", "button-large");
+      
+      // Apply the selected size class
```

Comment:
We only need to change one element in this function

Reply:
Oh, that's not my file, it's Nonty's, I think I might have accidentally changed its formatting and then included it into my commit. Not sure what happened. Shall I try to uncommit and unstage it somehow? Or can I remove it from my pull request some other way?